build.gradle.kts

```diff
@@ -62,6 +62,9 @@ allprojects {
                 showStackTraces = true
             }
         }
+        tasks.withType(Javadoc::class) {
+            options.encoding = "UTF-8"
+        }
```

Comment:
The encoding for javadocs is set in PlatformConfig.kt -> tasks.withType<JavaCompile>()

Reply:
The encoding was not set for Javadocs at all, which I moved into the PlatformConfig, as this could prevent a few UTF-8 issues

Reply:
Oic, I scrolled a bit down too far. Yeah, the method you added it fits. If we're at javadocs, we could utilize JEP 224 (generating javadocs with html5).